doc/source/parameters/dem/lagrangian_physical_properties.rst

```diff
@@ -30,10 +30,12 @@ In this subsection, gravitational acceleration, and the physical properties of t
 
       # If distribution type = custom
       set custom diameters                  = 0.001 , 0.0005
-      set custom probabilities              = 0.6   , 0.4
+      set custom volume fractions           = 0.6   , 0.4
 
       # If distribution type = normal
       set standard deviation                = 0.0
+      # If distribution type = normal or custom
+      set random seed distribution          = 1
```

Comment:
If you provide a seed to the pseudo-random number (PRN) generator, it is not a random seed. So I suggest:

```suggestion
      set prn seed = 1
```

Reply:
Like a random seed would be if you take the computer clock time or something like that

Reply:
In the insertion_info section of the parameter file, we have a parameter called "insertion random number seed". 

Knowing this, should I rename both parameters name : 
- prn seed distribution 
- prn seed insertion

Or I should just renamed this parameter to : 
distribution random number seed 

Note: There's also an other parameter called "insertion random number range". I just want to keep an uniformity between sections.

Reply:
or it could be "distribution seed" or something like this...
Random seed actually means taht the seed is indeed random. So this is a malapropism we have used too much in the code in the past